src/Microsoft.ML.AutoML/TrainerExtensions/SweepableParams.cs

```diff
@@ -115,6 +117,15 @@ public static IEnumerable<SweepableParam> BuildLightGbmParams()
             };
         }
 
+        public static IEnumerable<SweepableParam> BuildMatrixFactorizationParams()
+        {
+            return new SweepableParam[]
```

Comment:
What about the other sweepable params in MatrixFactorizationTrainer.cs?

* LossFunction
* Lambda
* Alpha
* C

Do we want to sweep over them?